diagnostic.md

```diff
@@ -184,5 +223,5 @@ with the class `.stretch` be at a _screen width_ of 550px?
 ```
 
 <!-- your answer starts here -->
-
+30px
```

Comment:
At a screen width of 550px, the `@media(min-width= 400px)` would apply. `.stretch` is 25% of width so 25% * 550 = 137.5px.